libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -87,6 +87,10 @@ def __ror__(
 
     """ --- Public API --- """
 
+    @property
+    def input_keys(self) -> Optional[List[str]]:
```

Comment:
In code: could we document what the purpose "input_keys" is for runnables?

Some questions...

I'm wondering whether input_keys are mostly for backwards compatibility purposes or if we think this is generally useful for UI / debugging. 

If it's the latter, then does it make sense to try to infer the schema instead and not worry about input_keys?

We could also try to auto-populate doc-strings of `invoke` / `batch` etc w/ parameters to help users discover what keys can be passed. (I tried that for RunnableLambda and it worked reasonably well)